openshift/auth.app.yaml

```diff
@@ -162,13 +152,22 @@ objects:
           - mountPath: /etc/fabric8/
             name: auth-serviceaccounts
             readOnly: true
+          - mountPath: /etc/fabric8/
+            name: auth-osoclusters
+            readOnly: true
         volumes:
         - name: auth-serviceaccounts
           secret:
             secretName: fabric8-auth-service-accounts
             items:
             - key: service-account-secrets
               path: service-account-secrets.conf
+        - name: auth-osoclusters
+          secret:
+            secretName: fabric8-auth-oso-clusters
```

Comment:
This is not consistent with the secrets configuration.  It should still be fabric8-auth-service-accounts, unless you create a new, separate configuration for fabric8-auth-oso-clusters.